packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -240,12 +237,12 @@ export const prepareLocalChainAccountKit = (
           // TODO #9211 lookup denom from brand
           if ('brand' in amount) throw Fail`ERTP Amounts not yet supported`;
 
-          destination.chainId in agoricChainInfo.connections ||
-            Fail`Unknown chain ${destination.chainId}`;
-
           // TODO #8879 chainInfo and #9063 well-known chains
-          const { transferChannel } =
-            agoricChainInfo.connections[destination.chainId];
+          const agoricChainInfo = await chainHub.getChainInfo('agoric');
+          const { transferChannel } = await chainHub.getConnectionInfo(
```

Comment:
given that these each do remote calls to the same vat, we should have a convinience for getting both

Reply:
thought about it, but it's necessarily 2 round-trips: 1 to get the chainInfo, then you have to pick out the chainId and send it for the 2nd call